Messaging.md

```diff
@@ -379,25 +389,27 @@ See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 This type contains an `oracle_event` and a signature certifying its origination.
 See [the Oracle specifications](./Oracle.md#oracle-announcements) for more details.
 
-#### Version 0 `oracle_announcement`
+#### `oracle_announcement`
 
-1. type: 55332 (`oracle_announcement`)
-2. data:
+1. data:
+   * [`bigsize`: `version`]
```

Comment:
I'm not sure I understand, are these simple sub-types then? If so I don't get the point of a simple sub-type with a version number, wouldn't it make more sense to have a sibling sub-type with just one version?

Reply:
You're right updated